elements/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -413,35 +384,57 @@ export class TransactionPool {
 			}
 			const processableTransactions = txList.getProcessable();
 			const allTransactions = [...processableTransactions, ...promotableTransactions];
-			let firstInvalidTransaction: { id: Buffer; status: TransactionStatus } | undefined;
-			try {
-				await this._applyFunction(allTransactions);
-			} catch (error) {
-				const failedStatus = this._getStatus(error as TransactionFailedResponse);
-				firstInvalidTransaction = {
-					id: (error as TransactionFailedResponse).id,
-					status: failedStatus,
-				};
+			const veirfyResults = [];
```

Comment:
also in other paces below
```suggestion
			const verifyResults = [];
```